src/main/java/edu/kit/kastel/sdq/intelligrade/login/CefUtils.java

```diff
@@ -29,6 +29,27 @@ private CefUtils() {
         throw new IllegalAccessError("Utility Class");
     }
 
+    /**
+     * Reset the CEF Browsers cookies to remove the artemis login token
+     */
+    public static void resetCookies() {
+        if (browserClient == null) {
+            browserClient = JBCefApp.getInstance().createClient();
+        }
```

Comment:
This is unnecessary 

It's also unnecessary here https://github.com/kit-sdq/intelligrade/blob/de38e97ae095797964a4cf9231f455fbf4a5ec14/src/main/java/edu/kit/kastel/sdq/intelligrade/login/CefUtils.java#L39-L41 though that's on me :D

Reply:
okay. removed it. I just copied your code ^^